restructured-document/system-model.tex

```diff
@@ -0,0 +1,17 @@
+\section{System model}
+The core structure of the system is based on the Client/server architecture %(right?)
+
+\subsection{Client}
+The client creates a job, specifies the OS, the hardware requirements and the priority of the job. The client then sends their job to the server via CLI. \newline
+The client could also interrupt or pause and resume a job during it's execution.
+
+\subsection{Central server/Gateway machine}
+The server receives jobs from clients, and passes them on to a remote working machine (/computational node). \newline
+It then receives results from the computational node and sends them to the client once the task is fulfilled. \newline
```

Comment:
The machines at our clients' location use a common file system.
The job results will be written to that common file system and will therefore not need to be sent through our system.
We only need to transfer metadata like if the job completed successfully.